pkgs/applications/misc/ptask/tw-version.patch

```diff
@@ -0,0 +1,11 @@
+--- a/src/tw.c	2017-02-21 22:56:19.289037608 +0100
++++ b/src/tw.c	2017-02-21 22:57:04.726814256 +0100
+@@ -116,7 +116,7 @@
+ 	    || !strcmp(ver, "2.3.0")
+ 	    || !strcmp(ver, "2.4.0")
+ 	    || !strcmp(ver, "2.4.1")
+-	    || !strcmp(ver, "2.5.0"))
++	    || !strcmp(ver, "2.5.1"))
```

Comment:
I can see why we need this patch, but I would really opt for contributing this upstream as well... I'm not sure why there is still the `2.5.0` version in here... Maybe both work? Have you tested it?